.devcontainer/devcontainer.json

```diff
@@ -76,7 +76,9 @@
 				"ms-dotnettools.vscodeintellicode-csharp",
 				"ms-azuretools.vscode-bicep",
 				"EditorConfig.EditorConfig",
-				"ms-azuretools.azure-dev"
+				"ms-azuretools.azure-dev",
+				"GitHub.copilot",
+				"GitHub.copilot-chat"
```

Comment:
Is this supposed to be usable by anyone or just the team 😄

Reply:
Just because the extension is installed doesn't mean people have to use it :)